README.md

```diff
@@ -1,81 +1,87 @@
+the `location-of-instalation/bin` to the $PATH variable so the contents are available in your terminal. This is an easy
+way to spin up a node locally however you will need to get the `bech32` yourself because it is not included in the
+Daedalus files
+
+### Testnet-magic numbers 
+
+In few places you will have to provide proper network parameter `(--mainnet | --testnet-magic NATURAL)`
+
+- 2: Preview. `--testnet-magic=2`
+- 1: Preprod. `--testnet-magic=1`
+
+### Cardano node socket port
+
+If you choose to go with the local node you need to have the `CARDANO_NODE_SOCKET_PORT` env variable set to the path of
+the local node socket file.
+
+## Step 1: Set up wallet
 
 There are three ways to setup a wallet.
 
-- Create new wallet
-- Restore wallet using mnemonics.
-- Restore a wallet using existing keys (without mnemonics).
+- a) Restore wallet using mnemonics.
+- b) Restore a wallet using existing keys (without mnemonics).
+- c) Create new wallet
 
-### Create a new wallet
+### a) Restore wallet from mnemonic
 
-1. Grant script permission to run `chmod +x ./scripts/setup-new-wallet.sh`
-2. Setup new wallet by running `yarn setup-new-wallet`
-3. Request Test ADA at [https://docs.cardano.org/cardano-testnet/tools/faucet](https://docs.cardano.org/cardano-testnet/tools/faucet)\
-   NB transaction fee it's a variable amount, something around 190.000 lovelace
-4. Query tip\
-   `$ cardano-cli query tip (--mainnet | --testnet-magic NATURAL)`
-5. Install packages\
-   `$ yarn install`
-6. Copy .env.example to .end and edit it accordingly\
-   `$ cp .env.example .env`
-7. Create a cip72 metadata file.
-8. Launch **cip72-cli** and follow the instructions\
-   `$ yarn start`
-
-### Restore wallet from mnemonic
-
-1. Make sure you have `bech32` and `cardano-address` installed. You can test with `bech32 -v` and `cardano-address -v` or install using `cabal install bech32 cardano-address`
-2. Create `phrase.prv` file and fill it up with your mnemonic in space seperated format.
-3. Grant script permission to run `chmod +x ./scripts/restore-mnemonic-wallet.sh`
-4. Restore mnemonic wallet by running `yarn restore-mnemonic-wallet`
-5. Make sure your wallet has test ADA for transaction fee
-6. Query tip\
-   `$ cardano-cli query tip (--mainnet | --testnet-magic NATURAL)`
-7. Install packages\
-   `$ yarn install`
-8. Copy .env.example to .end and edit it accordingly\
-   `$ cp .env.example .env`
-9. Create a cip72 metadata file.
-10. Launch **cip72-cli** and follow the instructions\
-    `$ yarn start`
-
-### Restore a wallet using existing keys
+1. Create `phrase.prv` file and fill it up with your mnemonic in space seperated format.
+2. Grant script permission to run `chmod +x ./scripts/restore-mnemonic-wallet.sh`
+3. Restore mnemonic wallet by running `NETWORK=(preview|preprod|mainnet) LOCAL_NODE=(true|false) yarn restore-mnemonic-wallet`
+(Choose appropriate values of the NETWORK and LOCAL_NODE variables)
```

Comment:
This command is applicable only if the user chooses to use the local node instead of blockfrost. Moreover, the previous script will fail on querying the protocol parameter if something is not working correctly so I don't see a big value in this step TBH.